examples/Retrieval-Models.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_path = '/workspace/data/train/*.parquet'
</pre><p> </p><p>Can we add a link and some instructions about how to get this data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
will do